google-http-client/src/main/java/com/google/api/client/http/HttpResponseException.java

```diff
@@ -127,6 +129,16 @@ public final String getContent() {
     return content;
   }
 
+  /**
+   * Returns the HTTP response or {@code null} for none as an input stream.
+   *
+   * @since 1.28
+   */
+  public final InputStream getContentAsInputStream() {
+    if (content == null) return null;
```

Comment:
nit: google style guide always uses parentheses for if and other block statements

Reply:
Resolved